tests/unit/test.index.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+var mockery = require('mockery');
+var nodemock = require('nodemock');
+
+var reloadCSS = require('../../index');
+
+module.exports = {
+
+  'reloadCSS() should fail when missing client option': function(test) {
+    reloadCSS({
+      app: {},
+      srcPath: '...'
+    }).then(function(results) {
+      test.ok(false);
+      test.done();
+    }).catch(function(err) {
+      test.done();
+    });
+  },
+
+  'reloadCSS() should fail when missing app option': function(test) {
+    reloadCSS({
+      client: {},
+      srcPath: '...'
+    }).then(function(results) {
+      test.ok(false);
+      test.done();
+    }).catch(function(err) {
+      test.done();
+    });
+  },
+
+  'reloadCSS() should fail when missing srcPath option': function(test) {
+    reloadCSS({
+      client: {},
+      app: {}
+    }).then(function(results) {
+      test.ok(false);
+      test.done();
+    }).catch(function(err) {
+      test.done();
+    });
+  },
+
+
+  'reloadCSS() should make Firefox API calls to reload CSS': function(test) {
```

Comment:
The placement of variable definitions and mixed use of `regularVars` and `ALL_CAPS` variables in this function has me a bit confused. I'm not sure why certain things are treated like `CONSTANTS` when they're all local to this function's scope anyway.